src/Streamly/Internal/Data/Stream/IsStream/Unfold.hs

```diff
@@ -0,0 +1,31 @@
+-- |
+-- Module      : Streamly.Internal.Data.Stream.IsStream.Unfold
+-- Copyright   : (c) 2017 Composewell Technologies
+-- License     : BSD-3-Clause
+-- Maintainer  : streamly@composewell.com
+-- Stability   : experimental
+-- Portability : GHC
+
+module Streamly.Internal.Data.Stream.IsStream.Unfold
+    (fromStream)
+where
```

Comment:
A module only for this looks like an overkill. Can it be moved to IsStream.Generate instead?